api/organization/update.js

```diff
@@ -26,9 +26,14 @@ module.exports = async (req, res, ctx) => {
       return res.send({ success: false, message: INSUFFICIENT_PERMISSIONS })
     }
 
-    // add billing email to stripe
-    await ctx.stripe.updateCustomerEmail({ customerId: org.billingInfo.customerId, billingEmail })
-    await ctx.db.organization.updateEmail({ orgId: org.id.toString(), email: billingEmail })
+    if (billingEmail) {
+      // add billing email to stripe
+      await ctx.stripe.updateCustomerEmail({ customerId: org.billingInfo.customerId, billingEmail })
+      await ctx.db.organization.updateEmail({ orgId: org.id.toString(), email: billingEmail, publicallyGive })
+    }
+    if (publicallyGive) {
+      await ctx.db.organization.updatePublicallyGive({ orgId: org.id.toString(), publicallyGive })
+    }
```

Comment:
Same idea as my comment in the db file -- how to orgs that at one point turned on public giving turn it off? I expect this value would be falsy in that case, and so not update the DB. Once solved there should be a unit test covering this case.

Reply:
stick a fork in it, cause it's done baby